api/v1/apiserver_types.go

```diff
@@ -185,6 +198,25 @@ type APIServerDeploymentSpec struct {
 	// Template describes the API server Deployment pod that will be created.
 	// +optional
 	Template *APIServerDeploymentPodTemplateSpec `json:"template,omitempty"`
+
+	// The deployment strategy to use to replace existing pods with new ones.
+	// +optional
+	// +patchStrategy=retainKeys
+	Strategy *APIServerDeploymentStrategy `json:"strategy,omitempty" patchStrategy:"retainKeys" protobuf:"bytes,4,opt,name=strategy"`
+}
+
+// TyphaDeploymentStrategy describes how to replace existing pods with new ones.  Only RollingUpdate is supported
```

Comment:
copy paste error in the comment `TyphaDeploymentStrategy`

Reply:
thanks, fixed